### PR TITLE
Improve mint form

### PR DESCRIPTION
**Pull request by makadaw**
image

### CODE REVIEW

File: src/pages/Mint.tsx [hunk 1]
```diff
@@ -1,238 +1,275 @@
...
-            borderColor: 'rgb(0, 0, 0)',
-          }}
-        >
-          <span className="absolute inset-0 flex items-center justify-center text-black tracking-wide uppercase RobotoMono-Bold_font__226og">
-            <span>&nbsp;Mint&nbsp;</span>
-          </span>
-        </div>
-      </div>{' '}
+        <span className="text-base">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
```
Comment (deus42) [inline src/pages/Mint.tsx:19]: Let's put here something meaningful or remove

**Review by deus42 — COMMENTED** [review]
Looks good.
Let's move the Mint button to right and add margin from the top a bit.
We can add a preview of the upload to the left probably. But it can be done later

File: src/pages/Mint.tsx (hunk 1 above)
Comment (makadaw) [reply src/pages/Mint.tsx:19]: I have added Lorem ipsum to have some placeholder for text. This form looks much better not on full screen. Will add some simple description

Comment (deus42) [reply src/pages/Mint.tsx:19]: Please notice comment in main chat

**Pull request merged**